reactor-netty-http/src/main/java/reactor/netty/http/server/ConnectionInfo.java

```diff
@@ -94,12 +96,18 @@ static ConnectionInfo from(Channel channel, HttpRequest request, boolean secured
 
 	ConnectionInfo(SocketAddress hostAddress, String hostName, int hostPort,
 			SocketAddress remoteAddress, String scheme, boolean isInetAddress) {
+		this(hostAddress, hostName, hostPort, remoteAddress, scheme, isInetAddress, "");
```

Comment:
I wonder if we'd actually like to provide the means to tell the difference between a missing header and the content being empty. If so, this could be actually `null` instead of empty by default. WDYT @violetagg ?

Reply:
yeah may be we should return `null` if there is no header or `forwarded=false`